dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-reports-reconciliation-summary.controller.js

```diff
@@ -0,0 +1,46 @@
+const { getAllBanks } = require('../../../services/repositories/banks-repo');
+const { UTILISATION_REPORT_RECONCILIATION_STATUS } = require('../../../constants');
+
+/**
+ * @typedef {import('../../../types/db-models/banks').Bank} Bank
+ * @typedef {import('../../../types/utilisation-reports').UtilisationReportReconciliationSummaryItem} ResponseBody
+ */
+
+/**
+ * @param {Bank[]} banks
+ * @param {string} submissionMonth
+ * @return {ResponseBody[]}
+ */
+// eslint-disable-next-line no-unused-vars
+const generateResponseBody = (banks, submissionMonth) =>
```

Comment:
FWIW I have a slight preference for using language/constants like `generateUtilisationReportSummaries` or something here and describing the entities.

Also why do you need no-unused-vars? Isn't it used on line 34?

Reply:
I think I made this name general because it's internal to this file, so clear in context. It would be better to be more specific though, so I'll update.

The `submissionMonth` isn't currently used in this function (line 34 is in the `getUtilisationReportsReconciliationSummary` function) but will be once I get around to 1043!